README.md

```diff
@@ -64,13 +64,13 @@ Access a growing collection of Material UI–inspired components (Buttons, Slide
 - **Powerful Theming Inheritance**
   Define a theme at a parent level and let it automatically apply to child components without extra configuration, reducing repetitive code while maintaining consistent branding.
 
-## Disclaimer
+Panel Material UI is still fairly new—first released in May 2025. As with any young library, you might run into a few rough edges as we continue to shape and improve it.
 
-Panel Material UI is a relatively new project, having been first released in May 2025. As with any young project, there may be some pains as we continue to develop and improve the library.
+We’re already using it in production, so development is active and updates are ongoing.
 
-We are actively using this library in production environments, which means you can expect regular updates and improvements.
+Thanks for your support as we—and maybe you?—keep making Panel Material UI even better!
 
-We appreciate your understanding and support as we work to make Panel Material UI the best it can be!
+Want to get involved? [Contribute on GitHub](https://github.com/panel-extensions/panel-material-ui/blob/main/DEVELOPER_GUIDE.md) or share your feedback—we’d love to hear from you.
```

Comment:
I notice you've started using a lot of `-` instead of `.`. Is that on purpose?

Reply:
Not in this case, removing it.